docs/hub/datasets-adding.md

```diff
@@ -1,13 +1,86 @@
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/upload_files.png"/>
+</div>
+
+2. Drag and drop your dataset files and add a brief descriptive commit message.
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/commit_files.png"/>
+</div>
+
+3. After uploading your dataset files, they are stored in your dataset repository.
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/files_stored.png"/>
+</div>
+
+### Create a Dataset card
+
+Adding a Dataset card is super valuable for helping users find your dataset and understand how to use it responsibly.
+
+1. Click on **Create Dataset Card** to create a Dataset card. This button creates a `README.md` file in your repository.
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/dataset_card.png"/>
+</div>
+
+2. At the top, you'll see the **Metadata UI** with several fields to select from like license, language, and task categories. These are the most important tags for helping users discover your dataset on the Hub. When you select an option from each field, they'll be automatically added to the top of the dataset card.
+
+    You can also look at the [Dataset Card specifications](https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1), which has a complete set of (but not required) tag options like `annotations_creators`, to help you choose the appropriate tags.
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/metadata_ui.png"/>
+</div>
+
+3. Click on the **Import dataset card template** link at the top of the editor to automatically create a dataset card template. Filling out the template is a great way to introduce your dataset to the community and help users understand how to use it. For a detailed example of what a good Dataset card should look like, take a look at the [CNN DailyMail Dataset card](https://huggingface.co/datasets/cnn_dailymail).
+
+### Dataset Viewer
+
+The [Dataset Viewer](./datasets-viewer) is crucial to know what the data actually look like.
+It is generally enabled by default for any dataset, depending on the dataset structure.
+
+Please refer to the documentation on [Dataset Structure](./datasets-structure).
+
+## Using Git
```

Comment:
remove this part (already written below, and this part mentions "models", not "datasets")